packages/expo-intent-launcher/android/build.gradle

```diff
@@ -0,0 +1,72 @@
+buildscript {
+  repositories {
+    google()
+    jcenter()
+  }
+
+  dependencies {
+    classpath 'com.android.tools.build:gradle:3.2.1'
+  }
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'maven'
+
+group = 'host.exp.exponent'
+version = '1.0.0'
+
+// Upload android library to maven with javadoc and android sources
+configurations {
+  deployerJars
+}
+
+// Creating sources with comments
+task androidSourcesJar(type: Jar) {
+  classifier = 'sources'
+  from android.sourceSets.main.java.srcDirs
+}
+
+// Put the androidSources and javadoc to the artifacts
+artifacts {
+  archives androidSourcesJar
+}
+
+uploadArchives {
+  repositories {
+    mavenDeployer {
+      configuration = configurations.deployerJars
+      repository(url: mavenLocal().url)
+    }
+  }
+}
+
+android {
+  compileSdkVersion 26
+
+  defaultConfig {
+    minSdkVersion 21
+    targetSdkVersion 26
+    versionCode 1
+    versionName '1.0.0'
+  }
+  lintOptions {
+    abortOnError false
+  }
+}
+
+if (new File(rootProject.projectDir.parentFile, 'pubspec.yaml').exists()) {
+  apply from: project(':expo_core').file('../expo-core.gradle')
+} else if (new File(rootProject.projectDir.parentFile, 'package.json').exists()) {
+  apply from: project(':expo-core').file('../expo-core.gradle')
+} else {
+  throw new GradleException(
+      '\'expo-core.gradle\' was not found in the usual Flutter or React Native dependency locations. ' +
+          'This package can only be used in such projects. Are you sure you\'ve installed the dependencies properly?')
+}
+
+dependencies {
+  expendency 'expo-core'
+  expendency 'expo-errors'
+
+  api 'com.android.support:support-annotations:28.0.0'
```

Comment:
Is it ok to include support-annotations in v28 whereas in other expo modules/expoview we use 27.1.1 iirc?